From d6637a3e37322c31dee4369746750598ea3a3eb4 Mon Sep 17 00:00:00 2001
From: Sergey Dyasli <sergey.dyasli@citrix.com>
Date: Tue, 19 Mar 2019 10:22:06 +0000
Subject: [PATCH 2/8] microcode: rename tool to xen-microcode

This was requested to be done by upstream and is expected to be included
into v7 of Chao's series.

Signed-off-by: Sergey Dyasli <sergey.dyasli@citrix.com>
diff --git a/tools/misc/xen-ucode.c b/tools/misc/xen-ucode.c
index da668ca62a..6e836b46bb 100644
--- a/tools/misc/xen-ucode.c
+++ b/tools/misc/xen-ucode.c
@@ -15,8 +15,8 @@
 void show_help(void)
 {
     fprintf(stderr,
-            "xenmicrocode: Xen microcode updating tool\n"
-            "Usage: xenmicrocode <microcode blob>\n");
+            "xen-ucode: Xen microcode updating tool\n"
+            "Usage: xen-ucode <microcode blob>\n");
 }
 
 int main(int argc, char *argv[])
